### ISSUE TITLE
 Unexpected verification output for  EXR files

### ISSUE THREAD

**Issue by ederenn** [golemfactory/clay#3460]
#### Encountered on:
Linux, binaries 0.17.1 and 0.18.0 rc 
checked on testnet

Tasks with output in Golem set as EXR file are failing with ` Unexpected verification output for subtask`

Subtask is computed by provider but fails on requestor's verification 

```
2018-10-17 20:49:11 INFO     golem.network.transport.session     Closing 35.198.119.10:35480
2018-10-17 20:49:11 WARNING  golem.docker.task_thread            Task stderr:
Traceback (most recent call last):
  File "/golem/work/job.py", line 71, in <module>
    params.xres, params.yres)
  File "/golem/work/job.py", line 60, in run_img_compare_task
    output_filename_path=file_path)
  File "/golem/scripts/img_metrics_calculator.py", line 40, in compare_crop_window
    xres, yres)
  File "/golem/scripts/img_metrics_calculator.py", line 136, in _load_and_prepare_img_for_comparison
    cropped_img = Image.open(cropped_img_path)
  File "/usr/local/lib/python3.6/site-packages/PIL/Image.py", line 2590, in open
    % (filename if filename else fp))
OSError: cannot identify image file '/golem/work/tmp/output/ref_small_shark1002_20001.exr'

2018-10-17 20:49:11 WARNING  golem.task.taskthread               Task computing error
2018-10-17 20:49:11 WARNING  apps.blender                        Unexpected verification output for subtask 'b2dfc32e-d23c-11e8-8e81-f479dec3e9bf',
```

**Comment by shadeofblue** [comment]
@magdasta 

[golem.log](https://github.com/golemfactory/golem/files/2488848/golem.log)

a full log from an integration test that reproduces the same issue

**Comment by shadeofblue** [comment]
[ possibly resolved - there's an integration test that checks it ]

though, generally, there's some issue with integration tests on develop

**Comment by shadeofblue** [comment]
[ maybe if it's indeed fixed, the fix should be cherry-picked into 0.18 ]

**Comment by shadeofblue** [comment]
I believe it's fixed
